modules/ui/osmose_details.js

```diff
@@ -0,0 +1,199 @@
+import {
+    event as d3_event,
+    select as d3_select
+} from 'd3-selection';
+
+import { modeSelect } from '../modes/select';
+import { t } from '../util/locale';
+import { services } from '../services';
+import { utilDisplayName, utilEntityOrMemberSelector } from '../util';
+
+
+export function uiOsmoseDetails(context) {
+    let _error;
+    const unknown = t('inspector.unknown');
+
+
+    function issueString(d, type) {
+        if (!d) return unknown;
+
+        // Issue description supplied by Osmose
+        var s = services.osmose.getStrings(d.error_type);
+        return (type in s) ? s[type] : unknown;
```

Comment:
If the description is missing you may as well not show the Description section instead of showing "Unknown".